desc/grid.py

```diff
@@ -440,19 +450,18 @@ def __repr__(self):
             type(self).__name__
             + " at "
             + str(hex(id(self)))
-            + (
-                " (L={}, M={}, N={}, NFP={}, sym={}, is_meshgrid={},"
-                " node_pattern={}, coordinates={})"
-            ).format(
-                self.L,
-                self.M,
-                self.N,
-                self.NFP,
-                self.sym,
-                self.is_meshgrid,
-                self.node_pattern,
-                self.coordinates,
-            )
+            + " (L={}, M={}, N={}, NFP={}, NFP_umbilic_factor = {},\
+               sym={}, node_pattern={}, coordinates={})"
+        ).format(
+            self.L,
+            self.M,
+            self.N,
+            self.NFP,
+            self.NFP_umbilic_factor,
+            self.sym,
+            self.is_meshgrid,
```

Comment:
`is_meshgrid={}` lost in string format above. Can either add back or just remove `self._is_meshgrid` here.

Reply:
Ok, resolved.